bencher.krun

```diff
@@ -0,0 +1,29 @@
+execfile("luajit.krun", globals()) # inherit settings from the main experiment
+
+MAIL_TO = ["fsfod11@gmail.com"]
+
+PRE_EXECUTION_CMDS = [
+    "sudo systemctl stop apt-daily.timer",
+    "sudo systemctl --runtime disable apt-daily.timer", 
+    "sudo systemctl stop apt-daily-upgrade.timer",
+    "sudo systemctl --runtime disable apt-daily-upgrade.timer",
+    
+    "sudo systemctl stop systemd-timesyncd",
+    "sudo systemctl --runtime disable systemd-timesyncd",
```

Comment:
Looks like you found a lot of stuff that I didn't. Will raise a PR to add these to the warmup experiment.